README.md

```diff
@@ -99,22 +99,36 @@ Please note that the API is throttled, currently 200 requests per minute, per ac
 
 If the retries are exceeded, or other API error is returned, `alpaca_trade_api.rest.APIError` is raised.
 You can access the following information through this object.
-
 - the API error code: `.code` property
 - the API error message: `str(error)`
 - the original request object: `.request` property
 - the original response objecgt: `.response` property
 - the HTTP status code: `.status_code` property
 
-### REST.get_account()
-Calls `GET /account` and returns an `Account` entity.
-
-### REST.list_orders(status=None, limit=None, after=None, until=None, direction=None, nested=None)
-Calls `GET /orders` and returns a list of `Order` entities.
-`after` and `until` need to be string format, which you can obtain by `pd.Timestamp().isoformat()`
-
-### REST.submit_order(symbol, qty, side, type, time_in_force, limit_price=None, stop_price=None, client_order_id=None, order_class=None, take_profit=None, stop_loss=None, trail_price=None, trail_percent=None)
-Calls `POST /orders` and returns an `Order` entity.
+### API REST Methods
+
+| Rest Method                                      | Description                                                                                | 
+| --------------------------------                 | -------------------------------------------------------------------------------------- |
+| get_account()                                    |  Calls `GET /account` and returns an `Account` entity.|
```

Comment:
Maybe you don't need "Calls" and have columns "endpoint" and "result type"?

Reply:
done